packages/orbit-design-tokens/output/docs-tokens.json

```diff
@@ -15069,15 +15092,15 @@
     },
     "javascript": {
       "name": "tabBundleBasicBackground",
-      "value": "linear-gradient(80.91deg, #E13E3B 0%, #E87E09 100%)"
+      "value": "#FFFFFF"
```

Comment:
why are these changed?

Reply:
because technically background is white, and foreground (text) has the gradient, I adjusted it to reflect it's behaviour :)